src/main/java/eiteam/esteemedinnovation/commons/init/ContentModuleHandler.java

```diff
@@ -83,7 +88,7 @@ public static void init() {
         Side side = getSide();
         boolean isClient = side == Side.CLIENT;
         for (ContentModule module : modules) {
-            module.recipes(side);
+            //module.recipes(side);
```

Comment:
Uncomment this. It's deprecated but we can still call it for the time being so it's easier to locate where this function is used.